### PR TITLE
Add recipe for dask-yarn

### PR DESCRIPTION
**Pull request by jcrist**

### CODE REVIEW

File: recipes/dask-yarn/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    Dask-Yarn deploys Dask on YARN clusters, as are found in traditional Hadoop
+    installations. Dask-Yarn provides an easy interface to quickly start, stop,
+    and scale Dask clusters natively from Python.
+  doc_url: https://dask-yarn.readthedocs.io/
+  dev_url: https://github.com/dask/dask-yarn
+
+
+extra:
+  recipe-maintainers:
+    - jcrist
```
Comment (mrocklin) [inline recipes/dask-yarn/meta.yaml]: Can I ask to be listed as maintainer as well?

File: recipes/dask-yarn/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - dask_yarn.cli
+
+about:
+  home: https://github.com/dask/dask-yarn
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: Deploy dask clusters on Apache YARN
+  description: |
+    Dask-Yarn deploys Dask on YARN clusters, as are found in traditional Hadoop
```
Comment (mrocklin) [inline recipes/dask-yarn/meta.yaml]: Should perhaps be "*such* as are found"  I think I made this mistake a while ago and it has unfortunately propagated.

File: recipes/dask-yarn/meta.yaml (hunk 1 above)
Comment (jcrist) [reply recipes/dask-yarn/meta.yaml]: done

**Comment by jcrist** [comment]
Not sure why the windows build error https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.26065. Passed on circle, so it's not the whole recipe. The error looks like a conflict on installing the package:

```
conda.exceptions.UnsatisfiableError: The following specifications were found to be in conflict:
  - dask-yarn==0.2.0=py27_0
```

Any idea @mrocklin, @mariusvniekerk?

**Comment by mariusvniekerk** [comment]
I did notice that one of the upstream deps of skein (grpcio) does not have a conda-forge equivalent.

**Comment by jcrist** [comment]
> Alternatively, you could skip the windows builds and fix them in the feedstock?

Did just that. It *could* work on windows, but I haven't tested that yet and don't have a windows hadoop setup yet to test on anyway. Should hopefully be good to go now.

**Comment by mariusvniekerk** [comment]
I'll take a look at this in the morning, travisci has a pretty decent backlog that will probably take about 10 hours to clear

**Comment by mrocklin** [comment]
CI passes!

**Pull request merged**